HandySquad/Services/Implementations/SkillSetService.cs

```diff
@@ -17,49 +17,14 @@ public SkillSetService(ISkillSetReposiotry skillSetReposiotry,IMapper mapper)
         _mapper = mapper;
     }
     
-    // public async Task<List<SkillSet>> GetAllSkillSetsAsync()
```

Comment:
Is this commented out code really needed?

Reply:
nope